homeassistant/components/velbus/config_flow.py

```diff
@@ -82,3 +85,36 @@ async def async_step_user(
             ),
             errors=self._errors,
         )
+
+    async def async_step_usb(self, discovery_info: usb.UsbServiceInfo) -> FlowResult:
+        """Handle USB Discovery."""
+        await self.async_set_unique_id(
```

Comment:
Usually we use this unique id format for usb devices
https://github.com/home-assistant/core/blob/dev/homeassistant/components/zha/config_flow.py#L106